res/photon/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Photon Styling Examples</title>
+    <style>
+      body, html {
+        font: message-box;
+        font-size: 11px;
+      }
+      .row {
+        border-left: 2px solid var(--grey-30);
+        padding: 0 3em 2em;
+        margin: 3em 0;
+      }
+      .row pre {
+        margin: 2em 0;
+        display: block;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="row">
+      <pre>&lt;button type="button" class="photon-button"&gt;Photon Button&lt;/button&gt;</pre>
+      <button type="button" class="photon-button">Photon Button</button>
```

Comment:
Looking at how it's rendered, I see I didn't use an height (probably because I didn't want to in the first case I used it, for the service worker notice). But we should have a default one of 32px (not for this PR of course).